src/commands/set.json

```diff
@@ -89,17 +93,33 @@
                 "name": "condition",
                 "type": "oneof",
                 "optional": true,
-                "since": "2.6.12",
                 "arguments": [
                     {
                         "name": "nx",
                         "type": "pure-token",
-                        "token": "NX"
+                        "token": "NX",
+                        "since": "2.6.12"
                     },
                     {
                         "name": "xx",
                         "type": "pure-token",
-                        "token": "XX"
+                        "token": "XX",
+                        "since": "2.6.12"
+                    },
+                    {
+                        "name": "comparison-value",
+                        "type": "string",
+                        "token": "IFEQ",
+                        "since": "8.1.0",
+                        "optional": true,
```

Comment:
It doesn't need optional, because the whole "oneof" block is optional.

We want the syntax `[NX | XX | IFEQ comparison-value]`, not `[NX | XX | [IFEQ comparison-value]]`.

Reply:
noted! misinterpreted it before. Have made the change.